api/segments/models.py

```diff
@@ -204,6 +212,16 @@ def _check_percentage_split_operator(self, identity):
             <= float_value
         )
 
+    def _check_modulo_operator(self, value: typing.Union[int, float]) -> bool:
+        try:
+            divisor, remainder = self.value.split("|")
+            divisor = float(str(divisor))
```

Comment:
I'm not sure if there's a need to cast to string before casting to float? I think `.split()` will always return a list of strings?